README.md

```diff
@@ -33,7 +33,7 @@ resources:
 * `password`: *Required if `auth_method` = `password`.* Admin password.
 * `admin_key`: *Required if `auth_method` = `certs`.* Base64 encoded PEM.
 * `admin_cert`: *Required if `auth_method` = `certs`.* Base64 encoded PEM.
-* `resource_type`: *Required.* Resource type to operate upon (valid values: `deployment`, `replicationcontroller`, `job`).
+* `resource_type`: *Required.* Resource type to operate upon (valid values: `deployment`, `replicationcontroller`, `job`, `pheonix-job`).
```

Comment:
typo ... `phoenix` is the right spelling

hmm.. so that's not a kubernetes resource so i'm a bit unsure about this